.golangci.yml

```diff
@@ -129,38 +145,22 @@ linters-settings:
     # Enable to ensure that nolint directives are all used. Default is true.
     allow-unused: false
     # Disable to ensure that nolint directives don't have a leading space. Default is true.
-    allow-leading-space: true
+    allow-leading-space: false
     # Exclude following linters from requiring an explanation.  Default is [].
     allow-no-explanation: []
     # Enable to require an explanation of nonzero length after each nolint directive. Default is false.
     require-explanation: true
     # Enable to require nolint directives to mention the specific linter being suppressed. Default is false.
-    require-specific: true
-
-  gomodguard:
-    blocked:
-      # List of blocked modules.
-      modules:
-        # Blocked module.
-        - github.com/pkg/errors:
-            # Recommended modules that should be used instead. (Optional)
-            recommendations:
-              - errors
-              - fmt
-            reason: "This package is deprecated"
-        - github.com/elastic/beats/v7:
-            reason: "There must be no Beats dependency"
+    require-specific: false
```

Comment:
Does this just allow writing `//nolint` instead of `//nolint: staticcheck`? I like mentioning the specific linter but I don't care that much about it.

Reply:
Indeed, I'll change it to true.